internal/util/util_test.go

```diff
@@ -148,6 +148,43 @@ func (ts *UtilTest) ResolveWhenParentProcDirEnvSetAndAbsoluteFilePath() {
 	assert.Equal(ts.T(), "/var/dir/test.txt", resolvedPath)
 }
 
+func TestYAMLStringify(t *testing.T) {
```

Comment:
IMO, this is less readable than stringified JSON config. Shall we stick to JSON?